src/components/map/MapService.js

```diff
@@ -1530,20 +1530,39 @@ goog.require('ga_urlutils_service');
           return layer;
         },
 
+        flyToAnimation: function(ol3d, center, destination, defer) {
+          var scene = ol3d.getCesiumScene();
+          var camera = scene.camera;
+          var globe = scene.globe;
+          var carto = globe.ellipsoid.cartesianToCartographic(destination);
+          $http.get(gaGlobalOptions.apiUrl + '/rest/services/height', {
```

Comment:
Are you sure because I specify the zoom level 17 as parameter of the function. May be Cesium load the tiles it needs to make the calculation. I haven't verfiied but I had presumed.